Config.psm1

```diff
@@ -54,10 +54,11 @@ function Get-AvailableConfigOptions {
         @{"Name" = "install_maas_hooks"; "DefaultValue" = $false; "AsBoolean" = $true;
           "Description" = "If set to true, MAAS Windows curtin hooks will be copied to the image root directory."},
          @{"Name" = "compression_format";
-          "Description" = "Select between tar, gz, zip formats or any combination between these."},
+          "Description" = "Select between tar, gz, tar.gz or zip formats"},
```

Comment:
you can choose tar.zip and it will work

Reply:
Hello,

It fixes the cases where 
virtual_disk_format=RAW
image_type=MAAS
are selected along with different compression formats.
I will check the identation